### ISSUE TITLE
Whitespaces missing after using github integration

### ISSUE THREAD

**Issue by Utzel** [GitbookIO/integrations#665]
When you connect github and edit a file the line breaks don't get carried back up to gitbook again.

How its looking in the original:
![image](https://github.com/user-attachments/assets/091f4df4-802d-47b1-a560-eda6f11d12f1)

Downloaded code via the github link:
![image](https://github.com/user-attachments/assets/9fd5d59a-3277-4915-8f2a-964c93a1bd95)

If you now push the file back up to gitbook the empty lines between the boxes don't get carried over. The spacing in the code before the push were not changed in any way. This sadly also happens for lists, basically rendering the whole github link useless...
Website after github push:
![image](https://github.com/user-attachments/assets/c20709ee-cdf7-40d6-a34a-5c050ba17048)

**Comment by addisonschultz** [comment]
I think this is expected from the way expandable sections are rendered in the front end - we’re also working on updating the in-app styles to match the published version for this block as well. 

Can you try adding this in the whitespace to force the expandables into new lines? 
`&nbsp;`

**Comment by addisonschultz** [comment]
Thanks for pointing this out - basically, it sounds like the conversion of adding `&nbsp;` (non breaking space) gets converted to a new markdown line with a space (new line + `&#x20;`) 

But it sounds like neither option gives you a space between the expandable blocks 🤔 

Can you try adding a h3 tag with empty content (`&#x20;`) between the expandable blocks to see if that would work as a workaround for now?

**Comment by Utzel** [comment]
Yes, that could be the issue with '@nbsp;'. It also adds an extra whitespace to text lines which offsets them.
h3 was till a bit to wide, h6 seems to do the trick, thanks for the help :)

**Comment by addisonschultz** [comment]
One last thing to try - if you use the h6 with “content” in it - does it stay? 

The content can be a space: “ “.

**Comment by Utzel** [comment]
Hey, hope you had a nice Christmas :) 

Thanks for the suggestion, we tried it and while the spacing worked correctly it still breaks the formatting of the next line as in the previous post. I suppose this is what you meant? 

![image](https://github.com/user-attachments/assets/aded0930-3713-44e7-9c58-d61ee3700938)
![image](https://github.com/user-attachments/assets/f2700f0b-817a-4261-908d-04155e13fecd)

**Comment by addisonschultz** [comment]
@Utzel thank you! Does a simple new line not work in this case (with the unordered list)?
